README.md

```diff
@@ -1,8 +1,8 @@
 # Overview
-Synopsys Detect for Jenkins simplifies running Synopsys Detect in your Jenkins builds.
+Black Duck Detect for Jenkins simplifies running Detect in your Jenkins builds.
 
 # Build
-[![Build Status](https://travis-ci.org/jenkinsci/synopsys-detect-plugin.svg?branch=master)](https://travis-ci.org/jenkinsci/synopsys-detect-plugin)
+[![Build Status](https://travis-ci.org/jenkinsci/blackduck-detect-plugin.svg?branch=master)](https://travis-ci.org/jenkinsci/blackduck-detect-plugin)
```

Comment:
@devmehtasynopsys Lets just remove this all together (travis link). This is not operational

Reply:
Removed